### DISCUSSION TITLE
Declination data for Cassiopeia in the res/constellations directory

### DISCUSSION THREAD

**Discussion by juanjoitab** [Haruno19/starfetch#11] [General]
I see that there's this line in the JSON file for Cassiopeia:

"declination": "+77.6923447° to –48.6632690°"

This means that it spans declinations from 77.69 degrees North to 48.66 degrees South and this doesn't make sense. The most Southern declination coordinate in this constellation is in fact still in the Northern hemisphere, so I think this Southern boundary should have a positive declination.

Looking at the constellation boundary TXT file with the IAU, the most Southern boundary is at declination +46.6757545° North. However, the most Northern boundary is exactly the one in the JSON file at +77.6923447° North, so I guess the source could be traced back to a typing error.

**Comment by Haruno19** [comment]
Thank you for pointing this out!
I've probably messed up the value while creating the JSON file. I'll fix it right away.
Thank you!

**Comment by cescha2018** [comment]
Hi, I'm trying to install this software but when I do it appears next message:

g++ -std=c++17 src/starfetch.cpp -o starfetch
src/starfetch.cpp:9:10: fatal error: filesystem: No existe el archivo o el directorio
 #include <filesystem>
          ^~~~~~~~~~~~
compilation terminated.
makefile:10: fallo en las instrucciones para el objetivo 'starfetch'
make: *** [starfetch] Error 1

hope you can tell me how I can fix this issue. Thanks a lot.

**Reply by Haruno19** [reply]
Opened issue #21 to address this problem :)
